### PR TITLE
Corrected script for some item #10

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

File: src/map/status.cpp [hunk 1]
```diff
@@ -7038,6 +7150,16 @@ static unsigned short status_calc_matk(struct block_list *bl, struct status_chan
...
+	if (sc->data[SC_ALMIGHTY])
+		matk += 30;
+	if (sc->data[SC_ULTIMATECOOK])
+		matk += 30;
+	if (sc->data[SC_LIMIT_POWER_BOOSTER])
+		matk += sc->data[SC_LIMIT_POWER_BOOSTER]->val1;
+	if (sc->data[SC_MAGICCANDY])
+		matk += 30;
+	if (sc->data[SC_SKF_MATK])
+		matk += sc->data[SC_SKF_MATK]->val1;
```
Comment (Atemo) [inline src/map/status.cpp:7162]: If I'm not wrong you already defined them in `status_calc_matk` in the line above, I guess it give double bonus. Do you mind to check the bonus?

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -27149,3 +27149,123 @@ Body:
...
+      bonus bCon,3;
+  - Combos:
+      - Combo:
+          - 2020RTC_Imp_TW    # 400082
+          - 2021RTC_Headset_TW    # 400164
+    Script: |
+      .@r = getequiprefinerycnt(EQI_HEAD_TOP);
+      bonus2 bSubRace,RC_Player_Human,1+(2*.@r/3);
+      bonus2 bSubRace,RC_Player_Doram,1+(2*.@r/3);
+      bonus bDelayrate,-1+(-.@r/3);
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: src/map/status.cpp (hunk 1 above)
Comment (idk-whoami) [reply src/map/status.cpp]: Oh, I saw an example from SC_2011RWC_SCROLL

Thanks!

**Review by Atemo — APPROVED** [review]

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**